### ISSUE TITLE
The output of `setup` command is splitted sometimes

### ISSUE THREAD

**Issue by etolstoy** [rambler-digital-solutions/Generamba#4]
Sometimes one of the questions produced by `generamba setup` command is splitted in parts. This question is about choosing Xcode targets.

**Comment by DevAlloy** [comment]
Just tried `generamba setup` on `GenerambaSandbox` project. Sometimes targets represented in cut version:

```
Select the appropriate target for adding your modules (print the index):
rambaSandbox
1. GenerambaSandboxTests
```

Looks like problem in `ask`, because there are no problems with `puts` that text.

My Ruby version:
`ruby 2.0.0p645 (2015-04-13 revision 50299) [universal.x86_64-darwin15]`

**Comment by DevAlloy** [comment]
Issue resolved by updating Ruby to `ruby 2.2.1p85 (2015-02-26 revision 49769)` version

**Comment by etolstoy** [comment]
Let's close this issue until someone encounters it again.
